recipe/meta.yaml

```diff
@@ -83,9 +85,11 @@ requirements:
     - vs2015_win-64                      # [win]
   host:
     - ncurses  # [not win]
-    - libiconv
+    - libiconv # [osx]
     - libxml2
-
+    - llvm-openmp       # [osx]
```

Comment:
As you add this to host section, you most likely want to add it also to the 'run:' section. Or are you sure that for OSX there is no dependency to openmp required?

Reply:
Added in run section too.